lineman/app/components/group_page/discussions_card/discussions_card.scss

```diff
@@ -1,3 +1,11 @@
+.discussions-card{
+  @include cardNoPadding;
+}
+
+.discussions-card__header{
```

Comment:
actually... I'm not sure where it is.. but basically there are 3 or 4 main kinds of "padding on these sides of things" classes that I'm trying to group in utilities with the lmo-card- prefix. If what you're doing here is generic card padding that could be reused by other things consider making it a lmo-card-something